proctor-consumer/pom.xml

```diff
@@ -32,13 +32,6 @@
             <artifactId>proctor-common</artifactId>
             <version>${project.version}</version>
         </dependency>
-        <dependency>
-            <!-- for jasper-el -->
-            <groupId>${project.groupId}</groupId>
-            <artifactId>proctor-tomcat-deps</artifactId>
-            <version>${project.version}</version>
-            <type>pom</type>
-        </dependency>
```

Comment:
I cannot see or remember if this dependency is required. If we don't see usage of this, then it's probably safe to remove, it seems it was added in an early version of proctor

Reply:
Got it.